modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/visualization_modes/VisualizationModes.tsx

```diff
@@ -70,7 +71,21 @@ export default function VisualizationModes(props: IFDSViewSectionProps) {
 									className="px-0"
 									key={visualizationMode.visualizationModeId}
 								>
-									<Component {...props} />
+									<ClayLayout.ContentCol className="c-gap-4 cards-visualization-mode">
```

Comment:
The `cards-visualization-mode` className is used specifically to check cards container in Playwright tests. We should not reuse it here.